### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by howareyouman**
Added statistic for place and date/time.

### CODE REVIEW

File: src/main/java/com/w6/nlp/Parser.java [hunk 1]
```diff
@@ -59,25 +69,41 @@ public Response generateResponse(final String input) {
...
             );
+            
             sentenseWhere = DateTimeParser.parseDateAndTimeFromString(
                 sentence, 
                 locationTags
             );
-            sentenseWhat = violentVerbsParser.getAllViolentVerbs(parse);
-        
+            
+            int typeOfSentence = 1;
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/Parser.java]: Find out better name. for example weightOfSentense.

File: src/main/java/com/w6/nlp/Parser.java [hunk 2]
```diff
@@ -59,25 +69,41 @@ public Response generateResponse(final String input) {
...
+            if (!sentenseWhat.isEmpty())
+            {
+                ObjectsAndSubjects objAndSubj = GetDoerAndVictim.getSubjectAndObjectOfViolence(parse,sentenseWhat);
+                sentenseWho.addAll(objAndSubj.subjects);
+                sentenseWhom.addAll(objAndSubj.objects);
+                typeOfSentence = 2;
+            }  
+            
+            addValueToRatedArray(typeOfSentence, ratedWhere, sentenseWhere);
+            addValueToRatedArray(typeOfSentence, ratedWhen, sentenseWhen);
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/Parser.java]: I think logic with type of sentence should be for every entry. Location in violent sentence is more important that location in other sentence.

File: src/main/java/com/w6/nlp/Parser.java [hunk 3]
```diff
@@ -117,10 +145,84 @@ public Response generateResponse(final String input) {
...
+                );
+            }
+        }
+        ratedList.clear();
+
+        ArrayList<Pair<String, Double>> values = new ArrayList<>();
+
+        for(String string : statMap.keySet()){
+            Pair<Integer, Integer> val = statMap.get(string);
+            values.add(new Pair<>(string, (1.0 *val.second)/val.first));
```
Comment (towelenee) [inline src/main/java/com/w6/nlp/Parser.java:203]: You normalize it on number of time you find it in a text. I think if word appears multiple times it's better than word that apper one time.

Comment (howareyouman) [inline src/main/java/com/w6/nlp/Parser.java:203]: So. How to sort, though?

**Pull request merged**